runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -220,14 +220,14 @@ NNFW_STATUS nnfw_session::load_circle_from_buffer(uint8_t *buffer, size_t size)
   try
   {
     _model = onert::circle_loader::loadModel(buffer, size);
+    _coptions = std::make_unique<onert::compiler::CompilerOptions>(*_model);
```

Comment:
It's strange that model loading is failed by compile option. IMO, we need to move validation check.

Reply:
Yes, it is a part of https://github.com/Samsung/ONE/pull/9334/files#r905917895 I mentioned in this PR main comment.

Reply:
Anyway, I think it is better to enclose whole load statements for other possible exceptions.

Reply:
Anyway I agree with you that we need to move this into try block. I'll think about this later.